go/common/constants/constants.go

```diff
@@ -0,0 +1,10 @@
+package constants
+
+import "math/big"
+
+// TODO this package is a temporary structure to ensure code wide constants are gathered before being properly decoupled
+
+var (
+	DefaultGasPrice = big.NewInt(2000000000)
+	DefaultGasLimit = uint64(1025_000_000)
```

Comment:
I don't remember the context exactly, but whenever I tried to go above 1024 the sim test *might* have been breaking. It was a while ago but it was something something max block gas limit.